src/Microsoft.Security.Utilities.Core/PreciselyClassifiedSecurityKeys/SEC101_200_CommonAnnotatedSecurityKey.cs

```diff
@@ -22,26 +22,41 @@ public override IEnumerable<string> GenerateTruePositiveExamples()
             int count = 0;
             int attempts = 0;
 
-            while(true)
+            foreach (bool longForm in new[] { true, false })
             {
-                foreach (bool longForm in new[] { true, false })
+                while (true) 
                 {
                     char testChar = (char)('a' + attempts++);
-                    string example = IdentifiableSecrets.GenerateCommonAnnotatedTestKey(IdentifiableSecrets.VersionTwoChecksumSeed,
-                                                                                        "TEST",
-                                                                                        customerManagedKey: true,
-                                                                                        platformReserved: null,
-                                                                                        providerReserved: null,
-                                                                                        longForm: false,
-                                                                                        testChar);
 
-                    if (example == null) { continue; }
+                    if (testChar == '{')
+                    {
+                        int z = 10;

```

Comment:
>int [](http://example.com/codeflow?start=24&length=3)

why don't we just use this condition to break? i.e., remove the count. 

basically, we just scan a - z and halt once we hit the brace.